yt/visualization/base_plot_types.py

```diff
@@ -321,15 +321,17 @@ def _get_best_layout(self):
 
         return size, axrect, caxrect
 
-    def _toggle_axes(self, choice):
+    def _toggle_axes(self, choice, draw_frame=None):
         """
         Turn on/off displaying the axis ticks and labels for a plot.
 
         choice = True or False
```

Comment:
A minor comment but for consistency one should probably write `choice: boolean` here